.github/workflows/e2e.yml

```diff
@@ -4,29 +4,14 @@ jobs:
   e2e:
     if: github.event_name == 'deployment_status' && github.event.deployment_status.state == 'success'
     runs-on: ubuntu-latest
+
     steps:
-      - name: Dump GitHub context
-        run: echo "$GITHUB_CONTEXT"
-        env:
-          GITHUB_CONTEXT: ${{ toJson(github) }}
-      - uses: actions/checkout@v2
-      - name: Setup node
-        uses: actions/setup-node@v2
-      - name: Get yarn cache directory path
-        id: yarn-cache-dir-path
-        run: echo "::set-output name=dir::$(yarn cache dir)"
-      - uses: actions/cache@v3
-        id: yarn-cache # use this to check for `cache-hit` (`steps.yarn-cache.outputs.cache-hit != 'true'`)
-        with:
-          path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
-          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
-          restore-keys: |
-            ${{ runner.os }}-yarn-
-      - name: Install dependencies
-        run: yarn install
-      - run: yarn add @synthetixio/synpress
-      - name: 'Update apt'
-        run: sudo apt-get update
+      - name: Chown workspace
+        run: chown -R $(whoami) .
+
+      - name: Checkout
+        uses: actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8 # pin@v2
```

Comment:
mind if we leave documentation somewhere, maybe in `cypress/README.md`, or `.github/workflows/README.md` about what's going on in this workflow? as a fresh user it might be hard to understand – would be great to leave context about the fact that this is coming from the synpress docker recommendations

Reply:
Make sense! will add a documentation here